src/operations/mod.rs

```diff
@@ -4,20 +4,20 @@ mod identify;
 mod resize;
 
 use crate::{
-    arg_parsers::{CropGeometry, LoadCropGeometry, ResizeGeometry},
+    arg_parsers::{CropGeometry, IdentifyFormat, LoadCropGeometry, ResizeGeometry},
     error::MagickError,
     image::Image,
 };
 
-#[derive(Debug, Copy, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Can't implement `Copy` since I used a `Vec` in the `IdentifyFormat` struct, oops.